optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc

```diff
@@ -508,6 +508,10 @@ optaplanner.solver-config-xml::
 A classpath resource to read the solver configuration XML.
 Defaults to `solverConfig.xml`.
 If this property isn't specified, that file is optional.
+optaplanner.constraints-drl::
```

Comment:
This is documented in the Spring boot section. We need it copy paste into the Quarkus section too.

Reply:
Order wise, I suggest we follow the order of SolverConfig.java
So this property should come after optaplanner.solver.move-thread-count and before optaplanner.solver.termination.spent-limit

Reply:
That's an option of course; I tried to follow the order of fields in the autoconfiguration.

Reply:
Decision: respect the order of field in the autoconfiguration.

Reply:
ad the Quarkus section: I was wandering: how could I have missed the quarkus section of our documentation? The answer is quite prosaic: the section is empty so far.
I assigned the https://issues.redhat.com/browse/PLANNER-2284 I filed to myself; not in scope of this PR, though.